circle.yml

```diff
@@ -30,6 +30,8 @@ dependencies:
 
 test:
   override:
+    # Check whether the committer forgot to run `go generate`.
+    - docker run cockroachdb/cockroach-dev shell "go generate ./... && git ls-files --modified --deleted | diff /dev/null -"
```

Comment:
Or you can add cockroach.sh to .gitignore (with a comment about when this occurs).